src/Disciplina/Async.hs

```diff
@@ -0,0 +1,19 @@
+{- | This module is an alias for lifted-async stuff.
+
+Among the project we use monad stack consisting of @ReaderT ctx m@,
+and thus tend to use 'Control.Concurrent.Async.Lifted.Safe' module for
+threading because it perfectly fits for IO-based monad stacks which have no
+state embeded into them, i.e. @StM m a ~ a@.
+(Among alternatives there are 'Control.Concurrent.Async' for 'IO' and
+'Control.Concurrent.Async.Lifted' for monad stack with embeded state which puts
+the burden of carrying state around on its user)
```

Comment:
Another alternative is `UnliftIO.Async`.

Reply:
@Martoon-00 
I've read documentation on `UnliftIO` (https://www.stackage.org/package/unliftio) and I think we should give it a shot. Async interface for it (https://www.stackage.org/haddock/lts-11.13/unliftio-0.2.7.0/UnliftIO-Async.html) is much simpler and doesn't use all these `Forall (Pure m)` constraints, which makes code simple and quicker to build.